CONTRIBUTING.rst

```diff
@@ -67,11 +67,10 @@ With respect to code standards, we follow `PEP8`_ and the `Google Style Guide`_.
 .. _`Google Style Guide` : https://google.github.io/styleguide/pyguide.html
 
 
-Licensing
+Licensing of contributions
 --------
 
-The License of this project is located in `LICENSE`_. By submitting a contribution to this project, you are agreeing that your contributions under this license and
-with this waiver of copyright interest.
+By submitting a contribution to this project, you are granting the project maintainers the ability to release your contribution under the project license located in `LICENSE`_. Along with this, you also waiver copyright interest.
```

Comment:
@rht This is not an area of expertise of mine, but it is my understanding that this is a implicit contributor's agreement rather than explicit, which is an industry standard until you have a code base that is well adopted enough, which you go explicit via explicit signing of documents to contribute.... if software projects did this in the early days of projects no one would contribute.

Putting that aside, @dmasad and I chatted about it during the meeting and decided to keep the same language just add a little clarification of wording that made it a little hard to read.